### PR TITLE
Add ZGRC_KEY_009

### PR DESCRIPTION
**Pull request by ewoutdenheijer**
added support for ZGRC_KEY_009

https://www.ledlampendirect.nl/draadloze-wandbediening-zigbee-rgbw-1-zone.html

### CODE REVIEW

**Comment by ewoutdenheijer** [comment]
shoul be ok now?

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -5313,6 +5313,33 @@ const converters = {
             }
         },
     },
+    ZGRC_KEY_009_cmdMoveHue: {
```
Comment (Koenkk) [inline converters/fromZigbee.js:5316]: Please try using the existing `command_move_hue` converter instead

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -5313,6 +5313,33 @@ const converters = {
...
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.movemode === 0) {
+                return {action: 'circle_release'};
+            } else {
+                return {action: 'circle_hold'};
+            }
+        },
+    },
+
+    ZGRC_KEY_009_cmdMoveColorTemp: {
```
Comment (Koenkk) [inline converters/fromZigbee.js:5328]: Could you change this to:

```js
command_move_color_temp: {
    cluster: 'lightingColorCtrl',
    type: 'commandMoveColorTemp',
    convert: (model, msg, publish, options, meta) => {
        const payload = {action: getProperty('color_temperature_move', msg, model)};
        if (msg.groupID) payload.action_group = msg.groupID;
        return payload;
    },
},
```

File: converters/fromZigbee.js [hunk 3]
```diff
@@ -5313,6 +5313,33 @@ const converters = {
...
+        cluster: 'lightingColorCtrl',
+        type: 'commandMoveColorTemp',
+        convert: (model, msg, publish, options, meta) => {
+            return {action: 'W_hold'};
+        },
+    },
+
+    ZGRC_KEY_009_cmdMoveColorTemp_release: {
```
Comment (Koenkk) [inline converters/fromZigbee.js:5336]: Could you provide the herdsman debug logging of this  command?

To enable herdsman debug logging, see https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt/pull/3952

**Pull request closed without merging**